src/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -27,6 +27,24 @@
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedViewStream.cs" />
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedFileRights.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot' ">
```

Comment:
'uapaot' should not be necessary - this library does not have aot specific config.